Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityPointerProfile.cs

```diff
@@ -92,5 +92,19 @@ public SystemType PointerMediator
         {
             get { return pointerMediator; }
         }
+
+        [SerializeField]
+        [Implements(typeof(IMixedRealityPrimaryPointerSelector), TypeGrouping.ByNamespaceFlat)]
+        [Tooltip("")]
+        private SystemType primaryPointerSelector = null;
```

Comment:
It would be nice to document what this is, either in the tooltip or in the README.